section6/locales-currencies-tz_diff.html

```diff
@@ -245,6 +251,23 @@ <h1>IsSanctionedSimpleUnitIdentifier ( _unitIdentifier_ )</h1>
         1. Return *false*.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-issanctionedcompoundunitidentifier" aoid="IsSanctionedCompoundUnitIdentifier">
+      <h1>IsSanctionedCompoundUnitIdentifier ( _unitIdentifier_ )</h1>
+
+      <p>
+        The IsSanctionedCompoundUnitIdentifier abstract operation verifies that the given core unit identifier is among the compound units sanctioned in the current version of the ECMAScript standard, a subset of the Validity Data as described in UTS #35, Part 1, Section 3.11; the list may grow over time. This list contains only compound units in which either the numerator or the denominator is not itself a sanctioned simple unit. The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. If _unitIdentifier_ is in the following list, return *true*.
+        <ul>
+          <li>liter-per-100kilometers</li>
+          <li>mile-per-gallon-imperial</li>
```

Comment:
I believe for both here and `IsSanctionedSimpleUnitIdentifier` there should be a Table with the sanctioned units of each kind and the normative text should say:

```
1. If the value of _unitIdentifier_ is listed in Table n, return *true*.
2. Else, return *false*
```

Of course this can be done in a follow up PR.

Reply:
`listed in Table` is a common approach used in ECMA-262 and the table could be expanded to add description or categorize each item.

Reply:
I'll make this in a follow-up PR.  I'm drafting it locally.